Library/Homebrew/cmd/audit.rb

```diff
@@ -1187,7 +1198,7 @@ def audit_urls
         problem "Don't use #{$1}use_mirror in SourceForge urls (url is #{p})."
       end
 
-      if p =~ /\/download$/
+      if p =~ %r{/download$}
```

Comment:
Until you feel strongly about using a regular expression: It doesn't really buy as anything in this particular case, thus a `p.end_with?("/download")` seems preferable to me.